src/Shared/Recruit.Vacancies.Client/Infrastructure/Repositories/MongoDbVacancyReviewRepository.cs

```diff
@@ -102,5 +114,36 @@ public Task<List<VacancyReview>> GetForVacancyAsync(long vacancyReference)
                 .Find(filter)
                 .ToListAsync(), new Context(nameof(GetForVacancyAsync)));
         }
+
+        public async Task<int> GetApprovedFirstTimeCountAsync(string submittedByUserId)
+        {
+            var filterBuilder = Builders<VacancyReview>.Filter;
+            var filter = filterBuilder.Eq(r => r.SubmittedByUser.UserId, submittedByUserId) &
+                         filterBuilder.Eq(r => r.Status, ReviewStatus.Closed) &
```

Comment:
Is this criteria necessary after checking for manual outcome?

Reply:
Belt and braces. It felt right to include the actual status of the review as part of this query.  If a status was to be reverted (through a data fix for example) then it won't be included in this count.